### PR TITLE
Adding some troubleshooting detail for Kerberos connections

### PR DESCRIPTION
**Pull request by jlownie**
##### SUMMARY
I spent a while troubleshooting an issue with Kerberos, I learned something that I think would be of benefit to other users.  This small PR contains that info.

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
WinRM connection using Kerberos

### CODE REVIEW

**Comment by samccann** [comment]
@jborean93 can you take a look at this since it's in the windows docs pls?

File: docs/docsite/rst/win_guide/windows_winrm.rst [hunk 1]
```diff
@@ -483,6 +483,8 @@ Kerberos is reliant on a properly-configured environment to
 work. To troubleshoot Kerberos issues, ensure that:
 
 * The hostname set for the Windows host is the FQDN and not an IP address.
+  * If you connect using an IP address you will get the error message `Server not found in Kerberos database`.
+  * To determine if you are connecting using an IP address or an FQDN run your playbook (or call the win_ping module) using the `-vvvv` flag.
```
Comment (jborean93) [inline docs/docsite/rst/win_guide/windows_winrm.rst]: 3 v's or above is needed, no need for 4 to show the hostname being connected to.

**Review by jborean93 — COMMENTED** [review]
Seems good to me, just a minor nit.

File: docs/docsite/rst/win_guide/windows_winrm.rst (hunk 1 above)
Comment (samccann) [inline docs/docsite/rst/win_guide/windows_winrm.rst]:
```suggestion
  * To determine if you are connecting using an IP address or an FQDN run your playbook (or call the ``win_ping`` module) using the `-vvv` flag.
```

**Review by samccann — APPROVED** [review]

**Comment by samccann** [comment]
Thanks @jlownie for the Ansible docs fix!

**Pull request merged**